README.md

```diff
@@ -66,6 +72,19 @@ All `Meter`s are reporting in `bytes`.
 
 * `process.threads`: The number of process threads as seen by the operating system.
 
+### ContextSwitchMetrics
+
+`ContextSwitchMetrics` reads process-level context switch information from `/proc/self/status`.
+
+> Please note that `procfs` is only available on Linux-based systems.
+
+* `process.voluntary.context.switch`: The accumulative number of voluntary context switches since application start.
+  Voluntary context switches happens when the thread is in waiting or blocked state, and scheduler switches control
+  to a different thread.
+* `process.nonvoluntary.context.switch`: The accumulative number of non-voluntary context switches since application start.
+  Involuntary context switches happens when the thread used all the timeslice and scheduler suspend it, and switches
+  control to a different thread.
```

Comment:
An involuntary context switch occurs when a thread consumed the whole time slice it was granted from the scheduler. The thread is suspended and control is switched to another thread.